src/panels/lovelace/editor/card-editor/hui-dialog-delete-card.ts

```diff
@@ -6,37 +6,32 @@ import {
   CSSResultArray,
   customElement,
   property,
-  query,
 } from "lit-element";
 
 import "./hui-card-preview";
 import "../../../../components/dialog/ha-paper-dialog";
 
 import deepFreeze from "deep-freeze";
 
-// tslint:disable-next-line
-import { HaPaperDialog } from "../../../../components/dialog/ha-paper-dialog";
 import { HomeAssistant } from "../../../../types";
 import { LovelaceCardConfig } from "../../../../data/lovelace";
 import { haStyleDialog } from "../../../../resources/styles";
 import { DeleteCardDialogParams } from "./show-delete-card-dialog";
+import { fireEvent } from "../../../../common/dom/fire_event";
 
 @customElement("hui-dialog-delete-card")
 export class HuiDialogDeleteCard extends LitElement {
   @property() protected hass!: HomeAssistant;
   @property() private _params?: DeleteCardDialogParams;
   @property() private _cardConfig?: LovelaceCardConfig;
-  @query("ha-paper-dialog") private _dialog?: HaPaperDialog;
 
   public async showDialog(params: DeleteCardDialogParams): Promise<void> {
     this._params = params;
     this._cardConfig = params.cardConfig;
     if (!Object.isFrozen(this._cardConfig)) {
       this._cardConfig = deepFreeze(this._cardConfig);
     }
-    if (this._dialog) {
-      this._dialog.open();
-    }
+    await this.updateComplete;
```

Comment:
If you are not going to do anything, there is no need to await the update. My idea was to call resize after updateComplete.

Reply:
ah gotcha. Sorry about that